nvflare/fuel/hci/zip_utils.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 import io
+import json
 import os
 from zipfile import ZipFile
+from nvflare.apis.job_def import JobMetaKey
 
 META_NAME = "meta.json"
 # A format string for the dummy meta.json
```

Comment:
maybe this `META_DATA` needs to update: "name" needs to change to `JobMetaKey.JOB_FOLDER_NAME.value`

Reply:
right that's simpler than what  I did

Reply:
let me try one more time.

Reply:
no, the template already has name in it. this is to cover the case meta.json exists but not have job name. 

Maybe I did not follow your comments